patches/chromium/fix_patches_for_webauthn.patch

```diff
@@ -0,0 +1,63 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Sential <sentialx@gmail.com>
+Date: Thu, 18 Mar 2021 21:05:59 +0100
+Subject: fix: patches for webauthn
```

Comment:
Can these patches get more descriptive subject lines and names? Perhaps the subject should specifically mention UI/modal, since it's not fixing WebAuthn exactly, it's providing UI support.